docs/guides/setting-url-wordpress-source-data..md

```diff
@@ -0,0 +1,114 @@
+# Setting the URL of the WordPress source of data
```

Comment:
Replace
"Setting the URL of the WordPress source of data"
with
"Setting the URL of the WordPress data source"